modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -13233,6 +13411,11 @@ input WhereProgram {
   """
   reference: WhereProgramReference
 
+  """
+  Matches the PI.
+  """
+  pi: WhereProgramUser
```

Comment:
WhereProgramUser allows specifying a role. I guess it's up to the user to not specify that for the PI, or to make sure it is `PI`? I think that is reasonable.

Reply:
Right, for this case matching on the program pi role and setting it to coi would be a complicated way to get an empty program list result.  The idea though is to use this same `WhereProgramUser` in the partner update mutation though.